src/sentry/quotas/redis.py

```diff
@@ -81,8 +81,12 @@ def get_quotas(self, project, key=None):
             ))
         return tuple(results)
 
-    def get_redis_key(self, key, timestamp, interval):
-        return '{}:{}:{}'.format(self.namespace, key, int(timestamp // interval))
+    def get_redis_key(self, key, timestamp, interval, shift):
```

Comment:
In retrospect, I don't think this should have ever been exported (this change was made in e9fbab1dcdc68fff5ee24b0a39b59fe35ce3b1ce) since this allows the storage abstraction to leak out of the implementation.

See getsentry/getsentry@e5b93513d4695e9f6c4be8c73685600a10030d75 for the motivation behind this change. Instead of this approach, we should add a `get_usage` method or something that allows you to fetch the usage that corrresponds to a `BasicRedisQuota` object (see above) rather than reaching in and communicating with the database directly.